scripts/loki_transform.py

```diff
@@ -104,12 +104,12 @@ def cli(debug):
               help="Generate offload instructions for global vars imported via 'USE' statements.")
 @click.option('--remove-derived-args/--no-remove-derived-args', default=False,
               help="Remove derived-type arguments and replace with canonical arguments")
-@click.option('--argument-array-shape/--no-argument-array-shape', default=False,
+@click.option('--derive-argument-array-shape/--no-argument-array-shape', default=False,
```

Comment:
Shouldn't this be `--no-derive-argument-array-shape` now as well? (Sorry, bit of a mouthful)